3rdparty/darknet/cmake/darknet.cmake

```diff
@@ -0,0 +1,149 @@
+cmake_minimum_required(VERSION 2.8)
```

Comment:
I think there is a possibility of placing the `CMakeLists.txt` to darknet upstream repo.
As one did in this PR: https://github.com/pjreddie/darknet/pull/6
Why not comment +1 at the PR?